README.md

```diff
@@ -417,7 +418,7 @@ Wraps the [SendEvent](https://docs.aws.amazon.com/connect-participant/latest/API
 The arguments are based on the [API request body](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_SendEvent.html#API_SendEvent_RequestSyntax) with the following differences:
 
 - Fields are in `camelCase`.
-- `ClientToken` cannot be specified.
+- `ClientToken`  is optional and can be used to ensure idempotency of the request.
```

Comment:
how does this work? 
the ACPS client is not taking ClientToken as an argument. Don't we need to update this and chatSession.sendMessage/Event()?
https://github.com/amazon-connect/amazon-connect-chatjs/blob/master/src/client/client.js#L240